random_menu_generator.rb

```diff
@@ -0,0 +1,55 @@
+# Random Menu Generator
+# This program takes user input and outputs a chosen number of unique random menu items
+
+# Introduce the generator and ask for input
+puts "Welcome to the Random Menu Generator.\n\n"
+puts "You get to pick a descriptor, a cooking style, and a type of food to combine into a random menu item."
+puts "The generator will make a list of unique menu items, so make sure to enter enough words in each category.\n\n"
+puts "Let's start with adjectives. Please enter a list of adjectives, separated by spaces:"
+
+# Gets input from the user and splits it on white space, then stores that in an array.
+adjectives = gets.chomp.split
+
+# Prompts user to give as many words in the second category as in the first
+puts "\nOkay, now give me some cooking styles.\nYou gave me #{adjectives.length} adjectives, so I recommend that many words here too."
+cooking_styles = gets.chomp.split
+
+# Prompts user to give as many words in the third category as in the smaller of the previous two
+puts "\nOkay, now give me some foods.\nYour smallest set of words so far is #{[adjectives.length, cooking_styles.length].sort[0]}, so I recommend that many words here too."
```

Comment:
I would advise against doing this level of sorting logic inside string interpolation. Since you need to use the `length_of_shortest` variable later, it would be much more readable if you set that variable before this output and then just reset the value on line 22 with the new `foods` array.